apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -286,11 +300,17 @@ export class BulkScriptAction extends BaseScriptAction {
           ' are okay to ignore.'
       });
     }
+
     this._ignoreHooksParameter = this.defineFlagParameter({
       parameterLongName: '--ignore-hooks',
       description: `Skips execution of the "eventHooks" scripts defined in rush.json. Make sure you know what you are skipping.`
     });
 
+    this._disableCacheFlag = this.defineFlagParameter({
```

Comment:
Should we only render this if the experiment is enabled and/or if the command itself isn't defined as disabling cache?